evap/student/templates/student_vote_questionnaire_group.html

```diff
@@ -35,6 +35,9 @@ <h4 class="{% if not forloop.first %} mt-3 {% elif contributor %} me-auto {% end
                         {% if field|is_choice_field %}
                             <div class="col-question col-lg-4 col-xl-5 d-flex justify-content-between{% if field.errors %} choice-error{% endif %}">
                                 <span class="me-1">
+                                    {% if not field.field.widget.attrs.counts_for_grade %}
```

Comment:
as far as I can tell this never occurs for the active evaluations in the test data, can you change the test data so that this shows up in for example Beautiful Data?